.rubocop.yml

```diff
@@ -233,35 +188,23 @@ Rails/Output:
 
 ################## Style #################################
 
-Style/Alias:
-  Enabled: false
-
 Style/AndOr:
-  Enabled: false
-
-Style/ArrayJoin:
-  Enabled: false
+  EnforcedStyle: 'conditionals'
 
 Style/AsciiComments:
-  Enabled: false
+  AllowedChars: ['–', '…']
```

Comment:
I don't particularly like this cop – it's very anglocentric, the guideline behind it, and not very unicode friendly (as we see here). Can we keep it disabled?

Reply:
Yeah, that's fine. The main useful thing I would expect it to do is 'prevent files getting horribly corrupted if they're loaded in another encoding' but that of course requires you're only using ascii in your _code_ too...